accounts-cluster-bench/src/main.rs

```diff
@@ -227,6 +229,264 @@ fn make_close_message(
     Message::new(&instructions, Some(&keypair.pubkey()))
 }
 
+#[derive(Clone, Copy, Debug)]
+pub enum RpcBench {
+    Version,
+    GetSlot,
+    GetMultipleAccounts,
+    GetProgramAccounts,
+    GetTokenAccountsByOwner,
+}
+
+#[derive(Debug)]
+pub enum RpcParseError {
+    InvalidOption,
+}
+
+impl FromStr for RpcBench {
+    type Err = RpcParseError;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s {
+            "get-slot" => Ok(RpcBench::GetSlot),
+            "get-multiple-accounts" => Ok(RpcBench::GetMultipleAccounts),
+            "get-token-accounts-by-owner" => Ok(RpcBench::GetTokenAccountsByOwner),
+            "version" => Ok(RpcBench::Version),
```

Comment:
Technically, version also has a `get` prefix in RPC: `getVersion`. Wdyt about removing all the `get`s from expected strings as well as from the `RpcBench` variants?

Reply:
ok, sure. I actually added Version because clippy doesn't like the Get prefix and I thought it might be useful as well, but I can just remove the 'get' prefix.